bdai_ros2_wrappers/bdai_ros2_wrappers/feeds.py

```diff
@@ -142,7 +144,7 @@ def close(self) -> None:
         super().close()
 
 
-class FramedMessageFeed(MessageFeed):
+class FramedMessageFeed(MessageFeed[MessageT]):
```

Comment:
Type annotations here aren't great, but I don't have enough `typing` features in Python 3.8 to annotate them properly.